### PR TITLE
Transactions explorer transform clean hist

### PR DESCRIPTION
**Pull request by jcbashdown**
This will use the new data to find the latest and the matching dashboard slug. Then for each metric in the latest it will post a separate record with the found slug to the output data set. It will overwrite any existing data for that slug/metric combination due to the id being identical.

In order to only ever write with the newest data we call is_latest_data on the target data set and compare the latest there to the value we will post. This moves is_latest_data out to util so it can be used as common code.

### CODE REVIEW

**Comment by jonnywyatt** [comment]
:+1:

**Comment by jcbashdown** [comment]
closing until I do some test fiddling.

**Comment by jcbashdown** [comment]
Test fiddling done

File: backdrop/transformers/tasks/latest_transaction_explorer_values.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    return data[0]
+
+
+def _get_stripped_down_data_for_data_point_name_only(
+        dashboard_config,
+        latest_data_points,
+        data_point_name):
+    """Builds up backdrop ready datum
+    for a single transaction explorer metric
+
```
Comment (mattrco) [inline backdrop/transformers/tasks/latest_transaction_explorer_values.py]: Can you fix the formatting?

Comment (jcbashdown) [inline backdrop/transformers/tasks/latest_transaction_explorer_values.py]: Sure, what specifically though? This passes my vim pep8 plugin and the docstring is pep257 compliant as far as I can work out.

Comment (mattrco) [inline backdrop/transformers/tasks/latest_transaction_explorer_values.py]: It's different to how most other docstrings are formatted in the codebase:

https://github.com/alphagov/backdrop/blob/master/backdrop/write/api.py#L69

Not a big deal, but consistency is nice.

**Pull request merged**